shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/security/DatasetSecurityService.java

```diff
@@ -63,7 +75,69 @@ public boolean hasRightOnStudy(Long studyId, String rightStr) {
 		}
         return commService.hasRightOnStudy(studyId, rightStr);
     }
-
+    
+    /**
+	 * Check that the connected user has the given right for the given study card.
+	 * 
+	 * @param studyCardId the study card id
+	 * @param rightStr the right
+	 * @return true or false
+     * @throws EntityNotFoundException 
+	 */
+    public boolean hasRightOnStudyCard(Long studyCardId, String rightStr) throws EntityNotFoundException {
+    	if (KeycloakUtil.getTokenRoles().contains("ROLE_ADMIN")) return true;
```

Comment:
On pourra en discuter pour les prochains dev (là je pense que c'est un peu tard :) )
https://github.com/fli-iam/shanoir-ng/wiki/Code-convention-and-best-practices